device/cuda/src/clusterization/clusterization_algorithm.cu

```diff
@@ -163,18 +163,14 @@ clusterization_algorithm::output_type clusterization_algorithm::operator()(
 
     // Create views to pass to cluster finding kernel
     const cell_container_types::const_view cells_view(cells_data);
-    vecmem::data::jagged_vector_view<unsigned int> sparse_ccl_indices_view =
```

Comment:
I am fully on board with this simplification in the CUDA code. But please put a few sentences into the description of the PR that mention this change/simplification.

Reply:
👍